pipelines/jobs/configurations/jdk17_pipeline_config.groovy

```diff
@@ -0,0 +1,203 @@
+                ],
+                test                : [
+                        nightly: [],
+                        weekly : ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf', 'sanity.external', 'special.functional']
+                ],
+                configureArgs       : [
+                        "openj9"      : '--enable-dtrace --enable-jitserver',
+                        "hotspot"     : '--enable-dtrace'
+                ]
+        ],
+        
+        x64LinuxXL  : [
+                os                   : 'linux',
+                arch                 : 'x64',
+                dockerImage          : 'adoptopenjdk/centos7_build_image',
+                dockerFile: [
+                        openj9  : 'pipelines/build/dockerFiles/cuda.dockerfile'
+                ],
+                test                 : 'default',
+                additionalFileNameTag: "linuxXL",
+                configureArgs        : '--with-noncompressedrefs --enable-dtrace --enable-jitserver'
+        ],
+
+        x64AlpineLinux  : [
+                os                  : 'alpine-linux',
+                arch                : 'x64',
+                dockerImage         : 'adoptopenjdk/alpine3_build_image',
+                test                : [
+                        // TODO: enable tests
+                        nightly: [],
+                        // release: ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf', 'sanity.external', 'special.functional']
+                        release: []
+                ]
+        ],
+
+        // Currently we have to be quite specific about which windows to use as not all of them have freetype installed
+        x64Windows: [
+                os                  : 'windows',
+                arch                : 'x64',
+                additionalNodeLabels: [
+                        hotspot: 'win2012&&vs2017'
+                ],
+                test                : [
+                        nightly: [],
+                        weekly : ['sanity.openjdk', 'sanity.perf', 'sanity.system', 'extended.system']
+                ]
+        ],
+        
+        x64WindowsXL: [
+                os                   : 'windows',
+                arch                 : 'x64',
+                additionalNodeLabels : 'win2012&&vs2017',
+                test                 : 'default',
+                additionalFileNameTag: "windowsXL",
+                configureArgs        : '--with-noncompressedrefs'
+        ],
+
+        aarch64Windows: [
+                os                  : 'windows',
+                arch                : 'aarch64',
+                crossCompile        : 'x64',
+                additionalNodeLabels: [
+                        hotspot: 'win2016&&vs2019'
+                ],
+                test                : [
+                        nightly: [],
+                        weekly : []
+                ]
+        ],
+
+
+        x32Windows: [
```

Comment:
Does that still make sense? i was under the impression that 32 bit on x64 is unmaintained.

Reply:
If it builds, I suggest we keep it for now

Reply:
We can make a formal support decision when 17 goes GA